packages/store/src/patterns/exo-makers.js

```diff
@@ -1,8 +1,18 @@
+/* global globalThis */
 import { objectMap } from '@agoric/internal';
 
 import { defendPrototype, defendPrototypeKit } from './exo-tools.js';
 
-const { seal, freeze } = Object;
+const { seal, freeze, defineProperty } = Object;
+
+// TODO Use environment-options.js currently in ses/src after factoring it out
```

Comment:
Yeah, this code never runs in the start compartment: even in a local (non-xsnap) worker, it's included in the kernel bundle, and evaluated into the kernel compartment. So it will never have a `globalThis.process` unless we explicitly copy it into the kernel-compartment globals.

I know we've accepted a pattern of using it to configure Endo/lockdown environments, but it weirds me out a bit to see `process` in otherwise pure code.

Reply:
Yeah, we discussed this at the time, and we did not come up with any alternative that seemed better enough that it was worth introducing a distinct convention. Weirded me out at the time too.

Btw, "we" === at least @kriskowal and I. Attn @kriskowal 

Once we factor out `environment-options.js` so it is independently usable, I hope it will mostly-encapsulate the messiness of this convention. But only "mostly" because it is still all openly observable state.

Resolving.

Reply:
See https://github.com/endojs/endo/issues/1266